index-el.html

```diff
@@ -567,7 +567,7 @@ <h2>Πως Σταματάμε μια Επιδημία</h2>
         
         Αυτή η τάση για αλληλεπίδραση με ανθρώπους που έχουν όμοιες προτιμήσεις <a href="https://www.sciencedirect.com/science/article/pii/S2468042720300191">εξανεμίζει την προστασία που παρέχουν οι μάσκες</a> επειδή <a href="https://github.com/aatishb/maskmath/blob/master/model/mathmodel.ipynb">όσοι δεν τις φορούν είναι λιγότερο πιθανό να ευεργετηθούν από κοντινούς τους ανθρώπους που φορούν</a>.
         <br/><br/>
-        Σύμφωνα με ανάλυση των <a href="https://www.sciencedirect.com/science/article/pii/S2468042720300191">David Fisman, Amy Greer, and Ashleigh Tuite</a>,
+        Σύμφωνα με ανάλυση των <a href="https://www.sciencedirect.com/science/article/pii/S2468042720300191">David Fisman, Amy Greer, και Ashleigh Tuite</a>,
         "η επίδραση των μασκών και άλλων προστατευτικών προσώπου στη μείωση της μετάδοσης του κορονοϊού είναι πιθανό να είναι μεγαλύτερη αν προβλεφθεί η διαθεσιμότητα τους σε μη-προνομιούχους πληθυσμούς".
       </span>
     </aside>
```

Comment:
εκφράζοτνας ---> εκφράζοντας in the paragram 'Το Ανθρώπινο Κράτος'

Reply:
paragraph*

Reply:
fixed, thanks!